src/explore-education-statistics-admin/src/components/editable/EditableBlockWrapper.tsx

```diff
@@ -35,7 +35,7 @@ const EditableBlockWrapper = ({
             </Button>
 
             <ModalConfirm
-              title="Delete section"
+              title="Remove data block?"
```

Comment:
`EditableBlockWrapper` wraps both data blocks and content text blocks. It would make more sense for this to say 'Remove block' instead. 

Also don't think you need to add question in the title either. You're already asking the question in the modal body itself.